docs/man/Dockerfile.5.md

```diff
@@ -29,198 +31,242 @@ A Dockerfile is similar to a Makefile.
+  it as holding externally-mounted volumes from the native host or from other
+  containers.
 
 **USER**
- -- **USER daemon**
- The USER instruction sets the username or UID that is used when running the
- image.
+  -- `USER daemon`
+  The USER instruction sets the username or UID that is used when running the
+  image.
 
-**WORKDIR**
- -- **WORKDIR /path/to/workdir**
- The WORKDIR instruction sets the working directory for the **RUN**, **CMD**, **ENTRYPOINT**, **COPY** and **ADD** Dockerfile commands that follow it.
- It can be used multiple times in a single Dockerfile. Relative paths are defined relative to the path of the previous **WORKDIR** instruction. For example:
- **WORKDIR /a WORKDIR b WORKDIR c RUN pwd** 
- In the above example, the output of the **pwd** command is **a/b/c**.
+**WRKDIR**
+  -- `WORKDIR /path/to/workdir`
+  The **WORKDIR** instruction sets the working directory for the **RUN**, **CMD**,
+  **ENTRYPOINT**, **COPY** and **ADD** Dockerfile commands that follow it. It can
+  be used multiple times in a single Dockerfile. Relative paths are defined
+  relative to the path of the previous **WORKDIR** instruction. For example:
+  `**WORKDIR /a WORKDIR b WORKDIR c RUN pwd**` 
+  In the above example, the output of the **pwd** command is **a/b/c**.
 
 **ONBUILD**
- -- **ONBUILD [INSTRUCTION]**
- The ONBUILD instruction adds a trigger instruction to the image, which is 
- executed at a later time, when the image is used as the base for another
- build. The trigger is executed in the context of the downstream build, as
- if it had been inserted immediately after the FROM instruction in the
- downstream Dockerfile.  Any build instruction can be registered as a
- trigger.  This is useful if you are building an image to be
- used as a base for building other images, for example an application build
- environment or a daemon to be customized with a user-specific
- configuration.  For example, if your image is a reusable python
- application builder, it requires application source code to be
- added in a particular directory, and might require a build script
- to be called after that. You can't just call ADD and RUN now, because
- you don't yet have access to the application source code, and it 
- is different for each application build. Providing  
- application developers with a boilerplate Dockerfile to copy-paste
- into their application is inefficient, error-prone, and
- difficult to update because it mixes with application-specific code.
- The solution is to use **ONBUILD** to register instructions in advance, to
- run later, during the next build stage.  
+  -- `ONBUILD [INSTRUCTION]`
+  The **ONBUILD** instruction adds a trigger instruction to the image, which is 
+  executed at a later time, when the image is used as the base for another
+  build. The trigger is executed in the context of the downstream build, as
+  if it had been inserted immediately after the FROM instruction in the
```

Comment:
This is not consistent. All commands/on-screen text needs to receive the same formatting.